docroot/themes/custom/govcms_uikit_starterkit/sass/styles.scss

```diff
@@ -8,16 +8,20 @@
 @import "reset";
 
 /**
- * GOV AU UI KIT 2.0 include.
- * You must run `npm install` within /uikit before this file will be available.
- */
-@import '../uikit/pancake/sass/uikit.scss';
-
-/**
-  Config file that contains centralized variables
+Config file that contains centralized variables
 */
 @import "config";
 
+// GOV.AU UI Kit
+//
+// Includes the UI kit modules that have been compiled during install into _uikit.scss.
+//
+// Configuration for the UI kit is included in _config.scss and can be used to override the defaults provided by the
+// UI kit modules.
+//
+// @see https://github.com/govau/uikit
+@import 'uikit';
```

Comment:
Moved this one after the config include so we can define configuration for UI kit for the configuration defined we need to run `gulp build` or `gulp build:dev` there isn't much documentation on the correct approach for this from govau but this works and let's us configure the uikit when it's compiled into `styles.css`.